src/app/shared/components/Breadcrumbs/BreadcrumbsItem/BreadcrumbsItem.tsx

```diff
@@ -366,7 +364,7 @@ const BreadcrumbsItem = (props: BreadcrumbsItemProps): JSX.Element => {
           ref={drop}
           className={`max-w-fit flex ${
             props.item.isFirstPath ? 'flex-shrink-0' : 'min-w-breadcrumb flex-1'
-          } cursor-pointer flex-row items-center truncate p-1 font-medium ${isDraggingOverClassNames}
+          } cursor-pointer flex-row items-center truncate py-1.5 px-3 font-medium ${isDraggingOverClassNames}
```

Comment:
Be careful with style changes, make sure you don't break other elements.

Before change:
![before](https://user-images.githubusercontent.com/57987155/215454937-b98a6994-686d-4080-9f56-ce095cd1173d.png)
After change:
![after](https://user-images.githubusercontent.com/57987155/215454940-ebac2194-f1c0-4863-836c-c432c8f35567.png)

Reply:
done